scripts/shared/resources/sm-global-cm.yaml

```
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+data:
+  use-nftables: "${NFTABLES}"
```

Since the quotes cause issues (and are required in the final YAML), I think it’s best to enforce them here:

```suggestion
  use-nftables: \"${NFTABLES}\"
```

This has the benefit of working without requiring specific quoting in the `NFTABLES` variable itself, and avoids misleading readers with unescaped quotes in this file (which seems to address the problem but doesn’t actually work).